src/app.js

```diff
@@ -41,3 +42,9 @@ mongoose.connect(dbConfig.DB_CONNECTION, {
 }).catch((error) => {
   console.log(`MongoDB connection Error ${error}`);
 });
+
+// vatsimAPI.getVatsimPilots().then((data) => {
```

Comment:
Probably don't need to keep this in the code, would be best to remove this

Reply:
Removed in latest commit